net_url/_urls.ts

```diff
@@ -0,0 +1,21 @@
+export class NetUrlUtils {
+    static HOMEPAGES = {
+        HTTPS: {
```

Comment:
Pulling out URL constants isn't unreasonable, however, it wouldn't make sense to break it down by protocol. It'd be better to just have global constants here (`const TWITTER = "...";` etc) and let them be imported as relevant.

Reply:
I don't agree with this opinion, it's better to have 'namespace' to understand that we're dealing with a url.

Reply:
I'm afraid I agree with @jeremy-rifkin here. Adding the HTTPS and HTTP distinction isn't helpful here. Semantically it doesn't matter what URI scheme they're in.

Reply:
I don't quite know why the reddit link isn't https, should probably be https.

I also don't think this file constitute a new top-level folder, and I think it would at least keep the regular naming convention (no leading underscore)

If we want this, I suggest we just export each link as its own constant, dropping the class and properties